ChangeLog.d/getrandom.txt

```diff
@@ -0,0 +1,2 @@
+Changes
```

Comment:
The version in the commit header and in the change log is not aligned with the one in the code: 2.15 vs 2.25.

Reply:
fixed.
it should be 2.25